src/analysis/souffle/taint.dl

```diff
@@ -32,50 +32,67 @@
 #ifndef SRC_ANALYSIS_SOUFFLE_TAINT_DL_
 #define SRC_ANALYSIS_SOUFFLE_TAINT_DL_
 
-.type AccessPath <: symbol
+#include "tags.dl"
 
-// A symbol representing a taint (or information flow) tag. These would be
-// strings of the form "secret", "public", "timestampInMs", etc. Just like
-// `AccessPath`, it would make sense to define an ADT in the long term.
-.type Tag <: symbol
+.type AccessPath <: symbol
 
 // Is this symbol an access path to some data?
 // (Useful for scoping when negation is involved.)
 .decl isAccessPath(accessPath: AccessPath)
 
-// Is this a Tag?
+// The set of all secrecy tags.
```

Comment:
It might make sense to move all of these to `tags.dl` as well. However, let us wait until Andrew merges #33